BannerlordTwitch/BLTAdoptAHero/BLTAdoptAHero.csproj

```diff
@@ -36,19 +36,19 @@
     </PropertyGroup>
     <ItemGroup>
         <Reference Include="0Harmony, Culture=neutral, PublicKeyToken=null">
-            <HintPath>..\..\..\..\..\..\..\Program Files (x86)\Steam\steamapps\common\Mount &amp; Blade II Bannerlord\Modules\Bannerlord.Harmony\bin\Win64_Shipping_Client\0Harmony.dll</HintPath>
+            <HintPath>C:\Program Files (x86)\Steam\steamapps\common\Mount &amp; Blade II Bannerlord\Modules\Bannerlord.Harmony\bin\Win64_Shipping_Client\0Harmony.dll</HintPath>
```

Comment:
Update from main to get my project file changes

Reply:
Yes, i will do that. Already updated it but not pushed yet. Nice change btw. It's make things easier ^^.